src/simulation_loop.py

```diff
@@ -0,0 +1,13 @@
+import vision.QAK.reading_data as rd
+
+if __name__ == "__main__":
+    base_path = "../data1500/void_1500-47/stairs0"
```

Comment:
Nie mogłem tego odpalić z tą scieżką, musiałem usunąć kropkę:
```suggestion
    base_path = "./data1500/void_1500-47/stairs0"
```

Reply:
zależy gdzie dałeś data1500 jeśli do katalogu głównego to musi być `../` a jeśli do katalogu `src/` to wtedy ok ale zostawiam `../`